### PR TITLE
Load available cloud build options when selecting the firmware version to build

### PR DESCRIPTION
**Pull request by blckmn**
Options can now be adjusted upon each release.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
something wrong with this. i can only auto-detect once.
https://youtu.be/8Zdcaw5bZdw

**Comment by blckmn** [comment]
> something wrong with this. i can only auto-detect once. https://youtu.be/8Zdcaw5bZdw

I don't think this is the cause of this problem. The change here is nothing to do with Autodetect, it merely moves the population of the options to "after" the selection of the firmware release, but before the slide out occurs.

**Comment by haslinghuis** [comment]
image

File: src/js/tabs/firmware_flasher.js
Comment (haslinghuis) [inline src/js/tabs/firmware_flasher.js:332]: Found the problem. We need to update line 611 as this was introduced in #3332

Comment (blckmn) [reply src/js/tabs/firmware_flasher.js:332]: Done.

Comment (blckmn) [reply src/js/tabs/firmware_flasher.js:332]: In reality I could probably simplify this a little and simple ask for the options for a given build key, setting the defaults as necessary. Save a round trip. I will do that in a subsequent PR.

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
Actually I'll need to make that change... otherwise the presets will get blown away

**Review by nerdCopter — APPROVED** [review]
* tested. seems to doing what is expected.

**Review by haslinghuis — APPROVED** [review]

File: src/js/tabs/firmware_flasher.js
Comment (haslinghuis) [inline src/js/tabs/firmware_flasher.js:336]: Nice :)

**Pull request merged**
